scanapi/config_loader.py

```diff
@@ -35,20 +35,12 @@ def construct_include(loader: Loader, node: yaml.Node) -> Any:
     extension = os.path.splitext(full_path)[1].lstrip(".")
 
     with open(full_path, "r") as f:
-        if extension in ("yaml", "yml"):
-            return yaml.load(f, Loader)
-        elif extension in ("json",):
-            return json.load(f)
-        else:
-            raise FileFormatNotSupportedError(f".{extension}", relative_path)
+        return yaml.load(f, Loader)
```

Comment:
I know is just 2 lines of code, but can we use a more meaningful name rather than just a single letter? :/
How about
```suggestion
        return yaml.load(file_hander, Loader)
```
instead? Don't forget to update line 37 if you accept this.

Reply:
If anything I'd pick `file` or `handle` which are both better than `f`, which one do you prefer?

Reply:
Maybe `handle`, as I'm not sure right now if `file` is a reserved name/module in python. That's why suggested `file_handler`.

Reply:
`file` is the name of a builtin in Python 2, but not in Python 3 :)

Reply:
Did not know that! Unfortunately I'm still stuck with some Python 2 projects :(